services/user.py

```diff
@@ -0,0 +1,51 @@
+from db.models import User
```

Comment:
According to the checklist, you should use `get_user_model()` instead of directly importing `User`. This is considered best practice.